### PR TITLE
Update change log for V4 preview 1 changes.

### PR DESCRIPTION
**Pull request by normj**
Update the change log with the changes recorded in the development tracker for the V4 preview 1 release.

### CODE REVIEW

File: changelogs/SDK.CHANGELOG.2024.md [hunk 1]
```diff
@@ -1,7 +1,40 @@
...
+    * Community [PR #3292](https://github.com/aws/aws-sdk-net/pull/3292) improving `AWSSDK.CopyTo` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3324](https://github.com/aws/aws-sdk-net/pull/3324) optimizing the AWS SigV4 signer. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3363](https://github.com/aws/aws-sdk-net/pull/3363) improving `AWSSDKUtils.ParameterAsString` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3365](https://github.com/aws/aws-sdk-net/pull/3365) improving `AWSSDKUtils.DetermineService` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3307](https://github.com/aws/aws-sdk-net/pull/3307) Optimizing `AWSSDKUtils.UrlEncode` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3425](https://github.com/aws/aws-sdk-net/pull/3425) Avoid allocating byte[] when converting MemoryStream to String. Thanks [Paulo Morgado](https://github.com/paulomorgado)
+
```
Comment (dscpinheiro) [inline changelogs/SDK.CHANGELOG.2024.md:35]: These extra lines (35-37) aren't needed, right? I don't see them in the `CHANGELOG.ALL` file.

File: changelogs/SDK.CHANGELOG.2024.md [hunk 2]
```diff
@@ -1,7 +1,40 @@
 ### 4.0.0.0-preview (2024-08-15 14:46 UTC)
+
+* AWSSDK.Extensions.NETCore.Setup (4.0.0.0)
+    * `DefaultClientConfig` in `AWSSDK.Extensions.NETCore.Setup` no longer extends from service client config base class `ClientConfig`. The service client config properties have been replicated on `DefaultClientConfig` using nullable value types to allow detecting when a value has been set on `DefaultClientConfig` when copying the values to the service client config being created for the service client.
+	* Packages has been mark as trim safe for Native AOT compilation.
```
Comment (dscpinheiro) [inline changelogs/SDK.CHANGELOG.2024.md:5]:
```suggestion
	* Package has been marked as trim safe for Native AOT compilation.
```

**Review by dscpinheiro — APPROVED** [review]

File: changelogs/SDK.CHANGELOG.2024.md (hunk 1 above)
Comment (normj) [reply changelogs/SDK.CHANGELOG.2024.md:35]: Removed

File: changelogs/SDK.CHANGELOG.2024.md (hunk 2 above)
Comment (normj) [reply changelogs/SDK.CHANGELOG.2024.md]: Fixed

**Review by peterrsongg — APPROVED** [review]

**Pull request merged**